app/models/user.rb

```diff
@@ -131,4 +134,12 @@ def recently_reached_by_user?(preg)
   def recently_called_by_user?(preg)
     preg.calls.any? { |call| call.created_by_id == id && call.recent? }
   end
+
+  def needs_password_change_email?
+    encrypted_password_changed? && persisted?
+  end
+
+  def send_password_change_email
+    UserMailer.password_changed(id).deliver
```

Comment:
Can you swap this `deliver` to `deliver_now`? (One less thing we'll have to fix when we migrate to rails 5!)